Airports/EGNT/Sector.txt

```diff
@@ -11,9 +11,19 @@ CIRCLE_SECTORLINE:NTTWR:EGNT:2.5
 DISPLAY:Only NTAPP:Only NTAPP:NTTWR
 DISPLAY:NTTWR:NTAPP:NTTWR
 
+CIRCLE_SECTORLINE:Only NTAPP:EGNT:5
+DISPLAY:East:East:NTAPP
+DISPLAY:Montrose:Montrose:NTAPP
+
+CIRCLE_SECTORLINE:Only NTAPP:EGNV:5
```

Comment:
This defines the same thing as above? If this is for Teesside, it needs to go in the Teesside file and be named Only NVAPP.

`DISPLAY:NTTWR:NTAPP:NVAPP` --> `DISPLAY:NTTWR:Only NTAPP:NVAPP`